tests/integration/test_vulnerability_detector/test_providers/test_providers_update_from_year.py

```diff
@@ -42,24 +45,24 @@
 configurations = load_wazuh_configurations(
     configurations_path, __name__, params=parameters, metadata=metadata)
 
+
 # fixtures
 @pytest.fixture(scope='module', params=configurations, ids=ids)
 def get_configuration(request):
     """Get configurations from the module."""
     return request.param
 
 
-# Tests
+# tests
```

Comment:
And this one.

Reply:
Done in 7fcdc7ebecdd546a4957814ec960f6304685d265